cli/module_registry.go

```diff
@@ -369,7 +369,7 @@ func (c *viamClient) uploadModuleFile(
 	var errs error
 	// We do not add the EOF as an error because all server-side errors trigger an EOF on the stream
 	// This results in extra clutter to the error msg
-	if err := sendModuleUploadRequests(ctx, stream, file, c.c.App.Writer); err != nil && !errors.Is(err, io.EOF) {
+	if err := sendModuleUploadRequests(ctx, stream, file, c.c.App.ErrWriter); err != nil && !errors.Is(err, io.EOF) {
```

Comment:
Justification: Most cli tools that I know of put progress bars / info on stderr instead of stdout.